mkt/files/helpers.py

```diff
@@ -92,8 +92,12 @@ def extract(self):
         Will make all the directories and expand the files.
         Raises error on nasty files.
         """
+        if self.file.status == mkt.STATUS_PUBLIC:
```

Comment:
Should we use `in mkt.LISTED_STATUSES` here? I'm thinking of the approved apps that aren't listed but still should exist in the public storage. I'd have to think harder about approved but private apps.